docs/sql-keywords.md

```diff
@@ -87,8 +87,7 @@ Below is a list of all the keywords in Spark SQL.
   <tr><td>DATA</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
   <tr><td>DATABASE</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
   <tr><td>DATABASES</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
-  <tr><td>DAY</td><td>reserved</td><td>non-reserved</td><td>reserved</td></tr>
-  <tr><td>DAYS</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
```

Comment:
according to https://www.postgresql.org/docs/current/sql-keywords-appendix.html , `unit-S` is not keyword.